lib/main/atom/views/outline/semanticView.ts

```diff
@@ -0,0 +1,64 @@
+import {NavigationTreeComponent} from "./navigationTreeComponent"
+import {NavigationTreeViewModel} from "./semanticViewModel"
+import {initialize} from "./semanticViewPane"
+
+export const SEMANTIC_VIEW_URI = "atomts-semantic-view"
+
+export interface SemanticViewOptions {
+  navTree?: NavigationTreeViewModel | null
+}
+
+export interface SemanticViewSerializationData {
+  data: {navTree: NavigationTreeViewModel | null}
```

Comment:
:warning: `data` should probably be `SemanticViewOptions`

Reply:
yes